packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -374,6 +377,28 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
       />
     );
   }, [fileSharing?.accept, fileSharing?.multiple, fileSharing?.uploadHandler, fileUploadButtonOnChange]);
+
+  /* @conditional-compile-remove(file-sharing) */
+  const activeFileUploads = useSelector(fileUploadsSelector).files;
+
+  const SelectedSendBox = (): JSX.Element => {
```

Comment:
Usually it's considered a bad idea to create a component inside another component. Could you mive it out of the ChatScreen?

Reply:
Will do!